Blizzard.IO.Core/Rpc/IRpcClient.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+
+namespace Blizzard.IO.Core.Rpc
+{
+    public interface IRpcClient
+    {
+        TRes Request<TReq, TRes>(TReq request);
+        Task<TRes> RequestASync<TReq, TRes>(TReq request);
```

Comment:
Maybe change the TRes and TReq to TRequest and TRespond?
```suggestion
        TRespond Request<TRequest , TRespond>(TRequest request);
        Task<TRespond> RequestASync<TRequest , TRespond>(TRequest request);
```